package.json

```diff
@@ -158,10 +158,27 @@
     "uuid": "^9.0.1"
   },
   "size-limit": [
+    {
+      "brotli": false,
```

Comment:
Hmmm this is interesting, alongside our treeshaking capabilities should we also look into brotli compressing our files in the future if we're not already. This comment is moreso to start a conversation for the future of some size reduction changes we could make.

Reply:
I don't think compressing our library would be particularly useful, since it would need to be decompressed by the library consumer when they consume it. It would be possible as something for them to do but would most definitely be a breaking change. Compressing the package would be a more beneficial optimization for an app, rather than the library.